conf.yaml

```diff
@@ -5,6 +5,7 @@ meta:
 repository:
     owner: galaxyproject
     name: galaxy
+    next_milestone: "16.04"
```

Comment:
yeah, I'm fine with manual updates since we're the only people who use p4 for the time being.